azure/durable_functions/tasks/new_uuid.py

```diff
@@ -11,7 +10,7 @@ def _create_deterministic_uuid(namespace_value: str, name: str) -> str:
     return str(uuid5(namespace_uuid, name))
 
 
-def new_uuid(context: DurableOrchestrationContext) -> str:
+def new_uuid(context) -> str:
```

Comment:
why are we stripping the type annotation here?

Reply:
It was causing a circular dependency because DurableOrchestrationContext imports this file and vice-versa. However, I just learned a way to fix this, so fixed!